api/Makefile

```
@@ -82,7 +104,7 @@ clean: docs-clean
 
 .PHONY: teardown
 teardown:
-	-$(pipenv) --rm
+	rm -rf .venv
```

Like we used to do with pipenv
I think here we want to use uv to resolve the location of the venv and remove it instead of resorting to rm.
`uv venv --remove` mean uv , use your resolution rules to find the venv for this project and remove it.
```suggestion
	$(UV) venv --remove
```